aten/src/ATen/functorch/DynamicLayer.h

```diff
@@ -113,6 +113,12 @@ TORCH_API Tensor unwrapIfDead(const Tensor& tensor);
 TORCH_API std::ostream& operator<<(std::ostream& os, const DynamicLayer& layer);
 TORCH_API std::ostream& operator<<(std::ostream& os, const std::vector<DynamicLayer>& dynamicLayerStack);
 
+// While a functorch grad transform is active, autograd.Function is disabled
```

Comment:
nit: Isn't it all functorch transforms? Since vmap is a problem if the input tensor requires grad

Reply:
Yup, you're right. Will update